### PR TITLE
Added tlg4072.tlg001.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Eutocius Commentarii in libros de sphaera et cylindro

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +49 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +30011 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg001/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3638)*

**Comment by AlisonBabeu** [comment]
This is just a small comment @sonofmun but should 
`<title>Archimède</title>` have a xml:lang="fre" tag?

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +49 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +30011 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg001/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3644)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +49 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +30011 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg001/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3647)*

File: data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,3607 @@
...
+<listBibl>
+ <biblStruct>
+ <monogr>
+  <title xml:lang="fre">Archimède</title>
+  <editor>
+	<persName>
+	 <name>Charles Mugler</name>
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg4072">Eutocius</author>
```
Comment (planatheisa) [inline data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml]: Should have `xml="lat"`.

File: data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,3607 @@
...
+  <imprint>
+   <publisher xml:lang="fre">Les Belles Lettres</publisher>
+	<pubPlace>Paris</pubPlace>
+	<date>1972</date>
+  </imprint>
+  <biblScope unit="volume">4</biblScope>
+ </monogr>
```
Comment (planatheisa) [inline data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml]: Scan missing.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +49 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +30011 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg001/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg001/tlg4072.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3669)*

**Pull request merged**